src/scss/web-components/_web-subscribe.scss

```diff
@@ -25,5 +25,48 @@ web-subscribe {
 
   .g-recaptcha {
     height: 78px;
+
+    &-- {
+      &placeholder {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        color: var(--color-action-bg);
+        align-items: center;
+        max-width: 304px;
+        height: 100%;
+        background-color: var(--color-mid-text);
+        border-radius: 5px;
+      }
+
+      &loader-container {
+        width: 100px;
+        background-color: var(--color-accent-bg);
+        border-radius: 20px;
+        padding: 2px;
+      }
+
+      &loader {
+        @keyframes loader {
+          0% {
+            transform: translateX(0);
+          }
+
+          50% {
+            transform: translateX(72px);
+          }
+
+          100% {
+            transform: translateX(0);
+          }
+        }
```

Comment:
Love the initiative, but I think it doesn't really fit the web.dev aesthetic. Can you give me a simple greyish or blue open circle spinner instead? 😅  Sorry! Might be good to even extract this to `scss/blocks` to have it reusable